### PR TITLE
JAVA-8405: reducing logging for tutorials-build-job

### PR DESCRIPTION
**Pull request by chaos2418**
Changes:
1) Updated maven-surefire-plugin in default-first and default-second profiles to add a 'logback.configurationFile' system property specifying the location of logback-config.xml in the root directory. As we are using logback in the root pom, adding this system property would reduce all the DEBUG level log messages.
2) Added logback-test.xml in src/test/resources of spring-boot modules. 
3) In some modules setting hibernate.show_sql to false so that SQL messages are not printed while executing tests.
4) Other minor updates to change INFO level log messages to DEBUG. 
5) Updated System.out.println messages to use LOGGER with DEBUG log level.

### CODE REVIEW

**Comment by chaos2418** [comment]
retest this please

**Comment by chaos2418** [comment]
retest this please

**Comment by asif-anwar** [comment]
retest this please

File: core-java-modules/core-java-lambdas/src/main/java/com/baeldung/java8/lambda/exceptions/LambdaExceptionWrappers.java [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(LambdaExceptionWrappers.class);
+
     public static Consumer<Integer> lambdaWrapper(Consumer<Integer> consumer) {
         return i -> {
             try {
                 consumer.accept(i);
             } catch (ArithmeticException e) {
-                System.err.println("Arithmetic Exception occured : " + e.getMessage());
+                LOGGER.error("Arithmetic Exception occured : {}", e.getMessage());
```
Comment (kwoyke) [inline core-java-modules/core-java-lambdas/src/main/java/com/baeldung/java8/lambda/exceptions/LambdaExceptionWrappers.java:17]: Let's use error(String msg, Throwable t); method here.
See https://www.baeldung.com/slf4j-log-exceptions

File: persistence-modules/spring-data-jpa-enterprise/src/test/java/com/baeldung/SpringContextTest.java [hunk 2]
```diff
@@ -3,12 +3,14 @@
...
 import org.junit.runner.RunWith;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.context.junit4.SpringRunner;
 
 import com.baeldung.boot.Application;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = Application.class)
+@TestPropertySource(properties = {"spring.jpa.show-sql=false "})
```
Comment (kwoyke) [inline persistence-modules/spring-data-jpa-enterprise/src/test/java/com/baeldung/SpringContextTest.java:13]: Maybe better to set this property in the application(-test).properties?

File: spring-caching/src/test/java/com/baeldung/ehcache/SquareCalculatorUnitTest.java [hunk 3]
```diff
@@ -20,21 +25,24 @@ public void setup() {
...
+            LOGGER.debug("Square value of {} is: {}", i, squaredCalculator.getSquareValueOfNumber(i));
         }
     }
 
     @Test
     public void whenCalculatingSquareValueAgain_thenCacheHasAllValues() {
         for (int i = 10; i < 15; i++) {
-            assertFalse(cacheHelper.getSquareNumberCache().containsKey(i));
-            System.out.println("Square value of " + i + " is: " + squaredCalculator.getSquareValueOfNumber(i) + "\n");
+            assertFalse(cacheHelper.getSquareNumberCache()
```
Comment (kwoyke) [inline spring-caching/src/test/java/com/baeldung/ehcache/SquareCalculatorUnitTest.java:37]: Keep the original formatting.

File: core-java-modules/core-java-lambdas/src/main/java/com/baeldung/java8/lambda/exceptions/LambdaExceptionWrappers.java (hunk 1 above)
Comment (chaos2418) [reply core-java-modules/core-java-lambdas/src/main/java/com/baeldung/java8/lambda/exceptions/LambdaExceptionWrappers.java]: yes updated now.

File: persistence-modules/spring-data-jpa-enterprise/src/test/java/com/baeldung/SpringContextTest.java (hunk 2 above)
Comment (chaos2418) [reply persistence-modules/spring-data-jpa-enterprise/src/test/java/com/baeldung/SpringContextTest.java]: Yes sure.

File: spring-caching/src/test/java/com/baeldung/ehcache/SquareCalculatorUnitTest.java (hunk 3 above)
Comment (chaos2418) [reply spring-caching/src/test/java/com/baeldung/ehcache/SquareCalculatorUnitTest.java]: Yep done. I need to update the code style in my IDE. will take care of that next time :)

**Comment by kwoyke** [comment]
retest this please

**Pull request merged**